sheet.go

```diff
@@ -110,6 +110,10 @@ func (s *Sheet) makeXLSXSheet(refTable *RefTable, styles *xlsxStyleSheet) *xlsxW
 		}
 		xRow := xlsxRow{}
 		xRow.R = r + 1
+		if row.IsCustom {
+			xRow.CustomHeight = true
+			xRow.Ht = fmt.Sprintf("%g", row.Height)
```

Comment:
Well, the way I have it, it only adds a height specifier if the user changes it from the default, thus making a simpler XML file (since only "non-default" values get written to the file).

If you do it the way you propose, then every row gets a row height explicitly, and now you have to go look up what the default was to determine if it's been changed.

Anyway - I tend towards a "less is more" approach, so my tendency would be to let defaults be defaults and only override explicitly when necessary, but if you guys want it the other way, that's fine by me.